requirements.txt

```
@@ -17,7 +17,7 @@ pandas>=0.25.0,<2.0.0
 tqdm>=4.33.0,<5.0.0
 
 # Internal models
-scikit-learn>=0.20.2,<0.22.0
+scikit-learn>=0.23.0,<0.25.0
```

Next version is going to be 1.0

```suggestion
scikit-learn>=0.20.2,<1.0.0
```

if it works for 0.20, why restrict it for >0.23?